scripts/build.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
```

Comment:
Can we please add some unit tests for this code? It looks that we should be able to validate a release version easily.